control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources_test.go

```diff
@@ -2349,3 +2349,109 @@ func TestReconcileAuthOIDC(t *testing.T) {
 		})
 	}
 }
+
+func initCondition(conditionType string, status metav1.ConditionStatus, reason, message string) *metav1.Condition {
+	return &metav1.Condition{
+		Type:    conditionType,
+		Reason:  reason,
+		Status:  status,
+		Message: message,
+	}
+}
+
+func initPod() corev1.Pod {
```

Comment:
The signature of the function does not match the expectations. I would rename it with a most appropriated name

Reply:
OK, I'll try to do better :)